ManikShrivastav.md

```diff
@@ -0,0 +1,45 @@
+---
+Name: “ManikShrivastav”
+Project: “Gesture Robot”
+---
+
+# Project Name
+Gesture Robot
+
+## Summary
+Gesture Robot is an electronic car controlled by the movement of circuit. It can be mostly used in real 
+life robot fights and car fights. It is a car that can be controlled according to the user’s movement of the 
+hands. The transmitter part will be attached to the hand of the user and its receiver will act as the 
+motherboard of the car. As the transmitter moves, it will send the signals to the receiver in the car and it 
+will turn and run the car according to the instructions.
+
+## Plan
+First of all, the chassis of car is going to be made. It will contain four motors and rubber wheels attached 
+to it. After that, the motors will be connected with motor shield driver. After that, the Arduino Nano is 
```

Comment:
> four motors and rubber wheels attached to it

Attached to what? I'm not sure if I missed something—what do you plan to make the frame out of?

Reply:
I am planning to make a frame our of Plywood which I already own and cutting them into a required shape is a simple job for me!